app/models/reader-store/reader-store.ts

```diff
@@ -294,12 +295,14 @@ export const ReaderStoreModel = types
           if (result.kind !== "ok") {
             throw new Error("READER_BOOKMARK_ADD_FAILED")
           }
+          logAnalyticsEvent("ToggleBookmarkAddBookmark", { url })
```

Comment:
Maybe add logs in screen is better in this case?

Reply:
The issue is that I would have to change the interface of all `onToggleBookmark()` to return `content` instead of `url`, such that I know which one is add and which one is remove.

Reply:
I have updated the onToggleBookmark callback prop to pass content instead, so now we can directly access the content to know `isBookmarked`, I will close this PR and let's continue in #215